android/shopify_buy_plugin/src/main/java/com/shopify/unity/buy/WebIntentFragment.java

```diff
@@ -0,0 +1,41 @@
+package com.shopify.unity.buy;
+
+import android.app.Fragment;
+import android.net.Uri;
+import android.support.customtabs.CustomTabsIntent;
+
+public class WebIntentFragment extends Fragment {
+
+    private boolean didShowIntent = false;
```

Comment:
After doing some exploration with `CustomTabsCallback`, `TAB_HIDDEN` is an event that is invoked when the user leaves the web view. This is something we don't want because we want to know when the user returns to the application, so we can make a check to see if the checkout has been completed.

For example, if the user presses the home button while the web view was shown, we will be receiving this event whilst the application is in the background.